zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/GroupEntityAddedHandler.java

```diff
@@ -55,7 +44,7 @@ public boolean handlesRecord(final Record<GroupRecordValue> record) {
 
   @Override
   public List<String> generateIds(final Record<GroupRecordValue> record) {
-    return List.of(String.valueOf(record.getKey()));
+    return List.of(String.valueOf(record.getValue().getEntityKey()));
```

Comment:
Is this correct? The member key (e.g. a user key) can't be the id of the member document because it can be a related to multiple groups.

Reply:
That's an excellent point, we'll need a new unique value. I wonder if concatenating the group and entity keys into a single "child" key is acceptable.

Reply:
How do we generate the key of a new group?

Reply:
We don't generate it in the exporter, we use the same key that is generated by Zeebe when the group is created. I think making a composite key, i.e. `groupKey-entityKey` would work here. We already do something similar [here](https://github.com/camunda/camunda/blob/78cb2c5deab1507b9dab8c5a5dbbafa81bd89a4a/webapps-schema/src/main/java/io/camunda/webapps/schema/entities/operate/listview/VariableForListViewEntity.java#L26-L28).